### ISSUE TITLE
Review Curriculum Advisors Handbook: Link to each role description?

### ISSUE THREAD

**Issue by maneesha** [carpentries/handbook-beta#134]
See https://carpentries-beta-handbook-preview.netlify.app/handbooks/curriculum_advisors

- [ ] Is all the content accurate?
- [ ] Do the links all work?
- [ ] Have you noticed any content duplicated in other places? 
- [ ] Any other feedback or suggestions

**Comment by tobyhodges** [comment]
Notes from initial review on 2024-10-22:

- Malformed link "(code of conduct)[https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html]." in https://carpentries-beta-handbook-preview.netlify.app/handbooks/curriculum_advisors#communication-and-collaboration-spaces
- Malformed link in https://carpentries-beta-handbook-preview.netlify.app/handbooks/curriculum_advisors#github-repositories
- In the _Onboarding_ section https://carpentries-beta-handbook-preview.netlify.app/handbooks/curriculum_advisors#onboarding, replace the first bullet point with the following, in bold:

```
- **Due to restricted capacity in the Curriculum Team, regular Curriculum Advisor Onboarding is currently suspended.**
  Individual Curriculum Advisory Committees may choose to recruit new members independently, making use of the process and any/all of the resources described below as needed. 
  If you are interested in joining an existing Curriculum Advisory Committee, we recommend that you contact the relevant group [via their mailing list on TopicBox](https://carpentries-beta-handbook-preview.netlify.app/handbooks/curriculum_advisors#mailing-list).
```

**Comment by kariljordan** [comment]
I'm reviewing the Curriculum Advisors Handbook, and here are my suggestions/questions:

- In the [meeting checklist](https://carpentries-beta-handbook-preview.netlify.app/handbooks/curriculum_advisors#meeting-checklist) there is a hyperlink that takes you up to the role description for Secretary, Chair, etc. for each of the bullet points. I'm wondering whether it's necessary. Is that for ease/accessibility?
- There is reference to a [randomizer](https://random.com/); FYI this domain is for sale. May need to consider another option if it's sold?
- The the [Curriculum Advisory Committee Consultation Rubric](https://docs.carpentries.org/topic_folders/lesson_development/cac-consult-rubric.html) leads to the old handbook. Should there be a page in the new handbook?

**Comment by maneesha** [comment]
Most of the points here are addressed in #182.
For the randomizer, I removed the link entirely and re-phrased the sentence to say assign roles randomly.

For the question about the links going back up to the role description, I will leave this question open for feedback.

**Comment by maneesha** [comment]
Addressed with #190
